testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/ServerGroupDeploymentManagedAttributeTestCase.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2020, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
```

Comment:
Unless this is being copied from somewhere else anything new to WildFly Core should really be contributed using an Apache license.

Reply:
license updated.